pkg/rollout/revision.go

```diff
@@ -0,0 +1,57 @@
+package rollout
+
+import (
+	"google.golang.org/api/run/v1"
+)
+
+// Annotations name for information related to the rollout.
+const (
+	StableRevisionAnnotation    = "rollout.cloud.run/stableRevision"
+	CandidateRevisionAnnotation = "rollout.cloud.run/candidateRevision"
+)
```

Comment:
suggestion: introduce a `lastUpdate` annotation.

imagine we see a service 50-50 split, no annotation, no tag. we're stuck, so inform user why.

Reply:
I'll work on this on a following PR. Is that ok?